setup.py

```diff
@@ -38,7 +38,7 @@ def get_version():
     test_suite='tests',
     install_requires=[
         'GitPython~=2.1.11',
-        'typing~=3.6.6',
+        'typing~=3.6.6 ; python_version<"3.7"',
```

Comment:
if it was part of the native library since 3.5, should we set this to <3.5 and use the native version if it's available?

Reply:
Good point. I decided to gate on 3.7 since that is the version that caused issues. Since it was only provisional in `3.5 & 3.6`, having it as a dependency was innocuous.

Reply:
Updated it anyways though.